packages/xo-web/src/xo-app/vm/tab-advanced.js

```diff
@@ -189,6 +192,15 @@ const PCI_ACTIONS = [
   },
 ]
 
+const SECUREBOOT_STATUS_MESSAGES = {
+  disabled: _('secureBootStatusNotEnforced'),
+  first_boot: _('secureBootStatusPending'),
+  ready: _('secureBootStatus'),
```

Comment:
This displays just "Secure Boot status". I think it's a mistake and that a `secureBootStatusReady` message is missing.